project/plugins.sbt

```diff
@@ -1,10 +1,11 @@
-
 resolvers ++= DefaultOptions.resolvers(snapshot = true)
 resolvers += Resolver.typesafeRepo("releases")
 
 addSbtPlugin("com.typesafe.play" % "interplay" % sys.props.get("interplay.version").getOrElse("1.3.4"))
 //addSbtPlugin("com.typesafe.play" % "play-docs-sbt-plugin" % sys.props.getOrElse("play.version", "2.5.10"))
 
+addSbtPlugin("com.typesafe" % "sbt-mima-plugin" % "0.1.8")
```

Comment:
According to [the docs](https://github.com/typesafehub/migration-manager/wiki/Sbt-plugin), the most recent version of MiMa sbt plugin is `0.1.13`.